### DISCUSSION TITLE
Ερωτηση για τη 2 και 3 Εβδομάδα

### DISCUSSION THREAD

**Discussion by p20stab** [courses-ionio/sw#1270] [Q&A] [answered]
Θα πρεπει το linux να εχει systemd για τη δευτερη εβδομαδα και να χρησιμοποιήσω ένα linux που δεν εχει για τη τριτη
ή
Θα πρεπει να βαλω systemd σε ενα linux που δεν εχει για την δευτερη εβδομαδα και να μην το χρησιμοποιήσω/διαγραψω για τη τριτη εβδομαδα?

**Comment by Angeloth1** [comment, marked as answer]
Καλησπέρα συνάδελφε για αυτήν την εβδομάδα δηλαδή την δεύτερη δεν έχει ιδιαίτερη σημασία το σύστημα το οποίο θα επιλέξεις. Σίγουρα ένα σύστημα linux σαν τα arch είναι το πιο βολικό (τουλάχιστον για εμένα). Αυτό που καλείσαι να υλοποιήσεις είναι:
- ~Μια άσκηση από την [λίστα](https://github.com/epidrome/dokey#software)~
- Δημιουργία ενός εγγράφου με την ομάδα σου σχετικά με το πως θα λειτουργείτε μπορείς, να δεις το ακόλουθο discussion #1234

Για την εβδομάδα 3 πρέπει να προσέξεις το σύστημα σου πρέπει να είναι customizable σε λειτουργικά ζητήματα. Μπορείς να δεις το ακόλουθο discussion #1237 όπου περιγράφεται ουσιαστικά από τους συναδέλφους @NickLitharis και @KonstantinosTourtsakis τα χαρακτηριστικά ενός δεκτού συστήματος. 

(Έγινε διόρθωση μετά από την παρατήρηση του καθηγητή με συγχωρείτε για την αναστάτωση)

**Reply by epidrome** [reply]
@Angeloth1 λένε κάπου οι οδηγίες για μια άσκηση από την λίστα;

**Reply by p20sami** [reply]
@epidrome Με βάση το σχόλιο σας αν εχω καταλάβει σωστά στο 2ο παραδοτεο αρκούσε η εγκατάσταση ενός systemd συστήματος και αν θελουμε μπορούμε να κανουμε μια άσκηση αν και δεν απαιτείται η άσκηση?

**Reply by epidrome** [reply]
@p20sami και όχι μόνο, διάβασε προσεκτικά την επιλεγμένη σωστή απάντηση και θα βρεις ποιο είναι το ζητούμενο

**Reply by p20sami** [reply]
@epidrome Με το σχόλιο εννοούσα για τη γραμμή εντολών μόνο καθώς το συνεργατικό κομμάτι του παραδοτέου το γνώριζα, απλώς ήθελα την διευκρίνιση μόνο για το μέρος της γραμμής εντολών, σας ευχαριστώ
